packages/expo-cli/src/commands/eject/clearNativeFolder.ts

```diff
@@ -104,6 +104,9 @@ export async function promptToClearMalformedNativeProjectsAsync(
       initial: true,
     }))
   ) {
+    if (isNonInteractive()) {
```

Comment:
It's unclear if any changes landed here will be published, I recommend opening here https://github.com/expo/expo/blob/main/packages/%40expo/cli/src/prebuild/clearNativeFolder.ts